web/ui/react-app/src/pages/rules/RulesContent.tsx

```diff
@@ -58,13 +59,16 @@ export const RulesContent: FC<RulesContentProps> = ({ response }) => {
             <Table bordered key={i}>
               <thead>
                 <tr>
-                  <td colSpan={3}>
+                  <td>
                     <a href={'#' + g.name}>
                       <h4 id={g.name} className="text-break">
                         {g.name}
                       </h4>
                     </a>
                   </td>
+                  <td colSpan={2}>
+                    <h4>Interval: {g.interval}s</h4>
```

Comment:
Would it make sense to use `humanizeDuration()` like we do below for the evaluation time? That way if the interval is multiple minutes (or even less than a second), it wouldn't be displayed as seconds.

Reply:
Yes it would! Added in [ff2d7bb](https://github.com/prometheus/prometheus/pull/12943/commits/ff2d7bb1d3136f4f157a968c12834acbd5300971)